viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1629,8 +1714,30 @@ export class Cognite3DViewer {
     offsetY: number,
     pickBoundingBox: boolean
   ): Promise<CameraManagerCallbackData> {
-    const intersection = await this.intersectModels(offsetX, offsetY, { asyncCADIntersection: false });
+    const modelIntersection = await this.intersectModels(offsetX, offsetY, { asyncCADIntersection: false });
 
+    // Find any custom object intersection closer to the camera than the model intersection
+    const customObjectIntersection = this.getCustomObjectIntersectionIfCloser(
+      offsetX,
+      offsetY,
+      modelIntersection?.distanceToCamera
+    );
+    if (customObjectIntersection == null || modelIntersection == null) {
```

Comment:
```suggestion
    if (customObjectIntersection === undefined || modelIntersection === null) {
```
It's a little unfortunate that the two intersection functions are inconsistent in using `undefined` and `null` as the failure-value. If you have time, would be great to make them return the same thing (preferably `undefined`)

Reply:
Fixed